README.md

```diff
@@ -8,20 +8,52 @@
 
 The [official docs for the LogDNA terraform provider](https://registry.terraform.io/providers/logdna/logdna/latest/docs) can be found in the Terraform registry.
 
-## Example Terraform Configuration
+Using the `logdna_alert` resource, a user can create a Preset Alert with a provided `name`, `email_channel`, `pagerduty_channel` and `webhook_channel`, delete a Preset Alert with a given `presetid` or update a Preset Alert using the `presetid` and `name`.
+
+Using the `logdna_view` resource, a user can create a View with provided `name`, `query`, `hosts`, `categories`, `tags`, `email_channel`, `pagerduty_channel`, and `webhook_channel`, delete a View with a given `viewid` or update a View using the `viewid` and `name`.
+
+Run `terraform init`, `terraform plan`, and `terraform apply`, refresh your browser and then navigate to the UI to see your updates.
+
```

Comment:
We should add links here to the many other files in the repo, especially the sample .tf files. I guess I fell like the readme file is supposed to act as a way to quickly understand the repo, and what all is in it.

Sorry I didn't think of this earlier.

If y'all agree, please add this paragraph:

"In addition to the simple examples below in this Readme, there are [sample `.tf` files](https://github.com/logdna/terraform-provider-logdna/tree/main/examples) in https://github.com/logdna/terraform-provider-logdna/examples."

Feel free to reword and tweak the link if I mucked it up, of course.

(I think the .go files are more for people who want to contribute, or see code for how our provide works? Do we need to mention/link to them?)

Reply:
I think linking the terraform examples makes sense (but not the .go files).

In addition to the examples provided below, sample .tf files can be found [here](https://github.com/logdna/terraform-provider-logdna/tree/main/examples).

Reply:
It looks good to me! I think, @mdeltito has been another reviewer - let's get his thoughts too